ci-operator/step-registry/cucushift/upgrade/eus/unpause-worker-mcp/cucushift-upgrade-eus-unpause-worker-mcp-commands.sh

```diff
@@ -53,18 +96,27 @@ function run_command() {
     eval "${CMD}"
 }
 
-
 if test -f "${SHARED_DIR}/proxy-conf.sh"
 then
     # shellcheck disable=SC1090
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
+if [[ "${PAUSED_MCP_NAME}" != "worker" ]]; then
+    pre_check_mcp
+    printf "\n"
+fi
 unpause
+printf "\n"
 if [[ $(oc get nodes -l node.openshift.io/os_id=rhel) != "" ]]; then
     echo "Found rhel worker, this step is supposed to be used in eus upgrade, skipping mcp checking here, need to check it after rhel worker upgraded..."
     run_command "oc get machineconfigpools"
     run_command "oc get node -owide"
 else
-    check_mcp
+    check_mcp ${PAUSED_MCP_NAME}
 fi
+
+printf "\n"
+if [[ "${PAUSED_MCP_NAME}" != "worker" ]]; then
```

Comment:
Per the discussion in the last meeting, `PAUSED_MCP_NAME` should allow user to input multiple mcp. The code here sounds like it has to be a single string.